gov2/rekognition/DetectLabels/README.md

```diff
@@ -0,0 +1,19 @@
+### DetectLabels/DetectLabels.go
+
+This example performs three tasks:
+
+1. Saves the image in an Amazon Simple Storage Service (Amazon S3) bucket with a "uploads/" prefix.
+1. Gets any ELIF information from the image and saves in the Amazon DynamoDB (DynamoDB) table.
```

Comment:
Also all items in the numbered list are number 1.

Reply:
See previous comment.